module/zfs/dmu.c

```diff
@@ -153,6 +161,34 @@ const dmu_object_byteswap_info_t dmu_ot_byteswap[DMU_BSWAP_NUMFUNCS] = {
 	{	zfs_acl_byteswap,	"acl"		}
 };
 
+/*
+ * Checking to see if a Direct IO write operation has updated the
+ * db_blkptr. We must pull the updated block pointer to get the
+ * current version of the block if a Direct IO write has occurred.
+ *
+ * The db_mtx must be held before calling this.
+ */
+static blkptr_t *
+dmu_buf_get_bp(dmu_buf_impl_t *db)
```

Comment:
The "dmu_buf_..." naming convention is normally used in dbuf.c. Since this is a private/static function in dmu.c, perhaps we should rename this to something like: get_bp_from_dbuf()

Reply:
The name has been updated in dmu_direct.c.